### DISCUSSION TITLE
✅ Add Yubikey support

### DISCUSSION THREAD

**Discussion by lilarcor** [isontheline/pro.webssh.net#790] [Ideas]
Hello, do you have the plan to add function of authenticating with Yubikey.

**Comment by isontheline** [comment]
Hello @lilarcor 👋

Thank you so much for your feedback 🙏
There is an open issue related to your request here : https://github.com/isontheline/pro.webssh.net/issues/697

I don't know however when I could implement Yubikey / FIDO2 support inside WebSSH as it's not a trivial task.
